ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -126,6 +130,11 @@ class DatasetToolArgs:
         positional=True,
     )
 
+    def __post_init__(self):
+        assert (
+            not self.upload_split or self.dataset_split
+        ), "Must specify dataset_split when using upload_split"
```

Comment:
Sorry I didn't catch this the first time, but why can't `upload_split` be equal to `dataset_split` by default if not specified?